README.md

```diff
@@ -24,7 +24,11 @@ CMR-GraphQL is an API developed by [NASA](http://nasa.gov) [EOSDIS](https://eart
 
 Before running the application you'll want to ensure that all necessary packages are installed by running:
 
-    npm install
+   npm install
+
+To install the edsc shared lint configuration:
```

Comment:
This step isn't necessary, it is included in the repo now and is covered by `npm install`